### PR TITLE
Carousel template for onwards upper

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
changes the onwards component from a static grid layout to a dynamic carousel layout

### Before
before

### After
after

## Why?
The small carousel layout was the winning variant for this component when AB tested

### CODE REVIEW

**Comment by jlieb10** [comment]
Awaiting any design requests/changes from Harry

**Review by nitro-marky — APPROVED** [review]
Looks great!

File: src/web/components/Onwards/OnwardsUpper.tsx [hunk 1]
```diff
@@ -70,6 +71,116 @@ const onwardsWrapper = css`
 	width: 100%;
 `;
 
+const headlinesContainer = (edition: Edition): string => {
```
Comment (lucymonie) [inline src/web/components/Onwards/OnwardsUpper.tsx:74]: These container functions all seem do the same thing - would it be worth abstracting this and passing a single function the type (eg 'headlines', 'sport', 'opinion' etc) and have it make the regional selection accordingly - maybe by accessing an object with the data?

**Review by lucymonie — APPROVED** [review]
Just one suggestion - otherwise, it looks good!

File: src/web/components/Onwards/OnwardsUpper.tsx (hunk 1 above)
Comment (jlieb10) [reply src/web/components/Onwards/OnwardsUpper.tsx]: great idea - please check now and lmk your thoughts :)

**Pull request merged**